python/pacemaker/_cts/watcher.py

```diff
@@ -9,7 +9,10 @@
 import time
 import threading
 
+from dateutil.parser import isoparser
```

Comment:
commit message really should say exerciser side, not server side

probably not worth fixing and triggering a new build, unless something else needs fixing. (Current CI failure is infra.)